BUILDING

```diff
@@ -14,15 +14,18 @@ Platform-specific notes
 
 - On OS X, the version of OpenSSL included with the operating
   system is outdated (0.9.8) and deprecated, and it is recommended
-  that spiped be built with an updated version of OpenSSL.  On OS X
+  that scrypt be built with an updated version of OpenSSL.  On OS X
   10.11 "El Capitan" and higher, OpenSSL was removed entirely.  After
   installing a newer version of OpenSSL, use
       CPPFLAGS="-I /path/to/openssl/headers"
-      LDADD="-L /path/to/openssl/lib"
-  to build spiped.
+      LDFLAGS="-L /path/to/openssl/lib"
+  to build scrypt.
 
   In particular, if you installed OpenSSL using homebrew, you may
-  add these lines to your $HOME/.profile file:
+  pass the relevant directories directly to ./configure:
+      ./configure CPPFLAGS="-I/usr/local/opt/openssl/include" LDFLAGS="-L/usr/local/opt/openssl/lib"
+
+  Alternately, you may with to add these lines to your $HOME/.profile file:
```

Comment:
Alternatively.  The use of "alternately" to mean "alternatively" is an American perversion.